recipes/lifecycle-hooks.md

```diff
@@ -49,9 +49,9 @@ class _MyComponentState extends State<MyComponent> {
   @override
   void didUpdateWidget(MyComponent oldWidget) {
     // this method IS called when parent widget passed new "props"
-    // unlike React this method IS callled _before_ the build
-    // unlike React method receives OLD "props"
-    // unlike React this method ISN'T called after setState()
+    // unlike React, this method IS called _before_ the build
+    // unlike React, this method receives OLD "props"
```

Comment:
Bdw I didn't understand this line

Reply:
You are right, initially it was `componentWillUpdate` on react side, but it was deprecated and I decided to use `componentDidUpdate`, so this explanation comment is redundant, but it worth replacing it with one mentioning that `didUpdateWidget` is called before `build`